### PR TITLE
AudioCell - set audio type to string

### PR DESCRIPTION
**Pull request by cfahlgren1**
We are adding support for more generic types for the SQL Console in the AudioCell [Internal PR](https://github.com/huggingface-internal/moon-landing/pull/11843#discussion_r1871635683).

### CODE REVIEW

**Comment by cfahlgren1** [comment]
@severo went ahead and made the width and height optional while we are at it

File: docs/source/openapi.json [hunk 1]
```diff
@@ -774,7 +773,7 @@
             "type": "integer"
           }
         },
-        "required": ["src", "height", "width"]
+        "required": ["src"]
```
Comment (severo) [inline docs/source/openapi.json:776]: Hmmm, note that it comes from

https://github.com/huggingface/dataset-viewer/blob/539b43775760f68f48d6df252ebef6e14c94edb6/libs/libcommon/src/libcommon/viewer_utils/asset.py#L20

It means that data coming from the API will always contain height and width. If you want to set them as optional on the Hub, maybe do so outside of the openapi.d.ts file.

Comment (cfahlgren1) [reply docs/source/openapi.json]: ahhh, makes sense. will revert this.

Comment (cfahlgren1) [reply docs/source/openapi.json]: wouldn't this be the same for audio type? 

We would expect that the API would only return this?

```python
              "enum": ["audio/wav", "audio/mpeg", "audio/ogg"]
```

and we need to make new type on moonlanding?

Comment (severo) [reply docs/source/openapi.json]: no, I checked in the code, it can be any extension after all

**Review by severo — APPROVED** [review]
Ok! Better now by relaxing the constraint, even if the API will always return height and width for now

**Pull request merged**